exonum/src/api/websocket.rs

```diff
@@ -33,20 +137,33 @@ pub(crate) struct Message(pub String);
 #[rtype(usize)]
 pub(crate) struct Subscribe {
     pub address: Recipient<Message>,
+    pub sub_types: Vec<SubscriptionType>,
 }
 
 #[derive(Message)]
 pub(crate) struct Unsubscribe {
     pub id: usize,
 }
 
+#[derive(Message)]
+pub(crate) struct UpdateSubscriptions {
+    pub id: usize,
```

Comment:
`usize` is platform dependent type and it shouldn't be used in network related code, use `u64` or `u32`.

Reply:
Agreed, but original code (like `Unsubscribe` structure) also uses `usize`, so I should either update all places or leave it "as is". And since this id is just an internal id of session (and never leave server), I don't see how it could cause an error.

Reply:
So it means that there was a flaw in original `Unsubscribe` code :) I think you may hardily fix it.

Reply:
Done (now `u64` used)